src/ngx_http_lua_kong_log.c

```diff
@@ -14,45 +14,78 @@
  * limitations under the License.
  */
 
-
 #include "ngx_http_lua_kong_common.h"
 
+#if (NGX_WIN32)
+/*
+** The dynamic log level feature is only supported on UNIX like system,
+** because we only call the function
+** void ngx_http_lua_kong_log_set_ready_for_dynamic_log_level();
+** in ngx_worker_process_cycle in the file src/os/unix/ngx_process_cycle.c.
+**
+** Excluding the technical view,
+** the Kong Gateway does not supoort the Windows OS.
+** 
+*/
+#error "The dynamic log level feature is only supported on UNIX like system"
+#endif
+
+/* -- Global variables for the dynamic log level feature ------------------- */
+
+/* the dynamic log level */
+static ngx_uint_t g_dynamic_log_level = NGX_CONF_UNSET_UINT;
+
+/* after this time, the dynamic log level will restore to original value */
+static time_t     g_dynamic_log_level_timeout_at; 
```

Comment:
Renaming to `ngx_lua_kong_dynamic_log_level_timeout_at` may be better.

Reply:
https://github.com/Kong/lua-kong-nginx-module/pull/60#discussion_r1178574727